### PR TITLE
Add deployment note for first `wasm-pack build --target web` example

### PR DESCRIPTION
**Pull request by maartin0**
Added a note to the example for a minimal build without a bundler (see: #1769)

### CODE REVIEW

File: guide/src/examples/without-a-bundler.md [hunk 1]
```diff
@@ -35,9 +35,18 @@ Otherwise the rest of the deployment magic happens in `index.html`:
 {{#include ../../../examples/without-a-bundler/index.html}}
 ```
 
+> **Note**: You cannot directly open `index.html` in your web browser due to [CORS][cors]
+> limitations. Instead, you can set up a quick development environment using
+> Python's built-in HTTP server:
+> ```sh
+> wasm-pack build --target web
+> python3 -m http.server 8080
```
Comment (stefnotch) [inline guide/src/examples/without-a-bundler.md:43]: Would it be possible to suggest some other kind of HTTP server? (One that's based on Node.js or Rust)

As for why I'm asking: Installing Python on Windows can be a bit finicky and does not always lead to a good user experience.

Comment (stefnotch) [reply guide/src/examples/without-a-bundler.md:43]: I don't see any changes, I'm guessing something got lost during a git command somewhere?

Comment (maartin0) [reply guide/src/examples/without-a-bundler.md:43]: Oops thanks for pointing that out; should be fixed now I think

Comment (stefnotch) [reply guide/src/examples/without-a-bundler.md:43]: Yay, nice

**Review by daxpedda — APPROVED** [review]

**Comment by daxpedda** [comment]
Thank you!

**Comment by maartin0** [comment]
No problem, do you want to close the issue as well?

**Pull request merged**